### ISSUE TITLE
Target framework mismatching: can't reference project SomeProject (.NETFramework,Version=v4.7) from Tests-Editor (.NETFramework,Version=v4.6) error when using alexzzzz's C#7+ integration

### ISSUE THREAD

**Issue by SugoiDev** [JetBrains/resharper-unity#430]
This is probably related to #429
I'll frequently see this in Rider's log.
It seems the csproj post processor either isn't doing its job, or what it does is being undone by another processor. See #429 for more reasoning.

**Comment by van800** [comment]
You may update TargetFrameworkVersion in Unity preferences - Rider.
Related https://github.com/JetBrains/resharper-unity/issues/417

**Comment by SugoiDev** [comment]
Of course, I forgot to mention that I'm already using 4.7 in the preferences.

![screenshot 2018 03 11-16 03 36](https://user-images.githubusercontent.com/7355630/37257491-fddcbcc0-2548-11e8-9f06-d2b5605161d3.png)


I don't have any other post processors for csproj files, so my guess is that the one from this integration is begin run out-of-order in relation to Unity's generator. I'm testing using the suggestion I mentioned in  #429 I remember dealing with a similar issue back when UnityVS still was a paid plugin and the project generation hooks were flimsy.

Will report back if this happens to fix this intermittent issue.
